INSTALLATION.md

```diff
@@ -213,6 +214,11 @@ When you've met the length requirements and added a tag, the "Save Post in Q&A"
 ![img/post.png](img/post.png)
 
 
+## Optional: Single Sign On
+
+You can [set up SAML Single Sign On](https://github.com/codidact/qpixel/wiki/Setting-up-SAML-Single-Sign-On).

```

Comment:
I'd reword it to something like "Please see our wiki for [detailed instructions](https://github.com/codidact/qpixel/wiki/Setting-up-SAML-Single-Sign-On) on setting up SAML Single Sign On.